controllers/twophase/types.go

```diff
@@ -61,206 +59,64 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}
 	chaos := _chaos.(v1alpha1.InnerSchedulerObject)
 
-	duration, err := chaos.GetDuration()
-	if err != nil {
-		r.Log.Error(err, "failed to get chaos duration")
-		return ctrl.Result{}, err
-	}
+	status := chaos.GetStatus()
+
+	targetPhase := status.Experiment.Phase
```

Comment:
If `status.Experiment.Phase`and `nextStartTime` both are empty, what will happen?

Reply:
`targetPhase` will be `Running`, as `chaos.GetNextStart().Before(now)` is true.